controllers/trial_job_controller.go

```diff
@@ -66,23 +68,25 @@ func (r *TrialJobReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	}
 
 	// Create a new job if necessary
-	if len(jobList.Items) == 0 {
-		// Insert a "sleep" between "ready" and the trial job
-		if ids := time.Duration(t.Spec.InitialDelaySeconds) * time.Second; ids > 0 {
-			for _, c := range t.Status.Conditions {
-				if c.Type == redskyv1beta1.TrialReady {
-					startTime := c.LastTransitionTime.Add(ids)
-					if startTime.After(now.Time) {
-						return ctrl.Result{RequeueAfter: startTime.Sub(now.Time)}, nil
-					}
+	if len(jobList.Items) > 0 {
+		return ctrl.Result{}, nil
+	}
+
+	// Insert a "sleep" between "ready" and the trial job
+	if ids := time.Duration(t.Spec.InitialDelaySeconds) * time.Second; ids > 0 {
+		for _, c := range t.Status.Conditions {
+			if c.Type == redskyv1beta1.TrialReady {
+				startTime := c.LastTransitionTime.Add(ids)
+				if startTime.After(now.Time) {
+					return ctrl.Result{RequeueAfter: startTime.Sub(now.Time)}, nil
```

Comment:
This was more of a cosmetic change and can be reverted if needed. Wanted to unwrap the code block by returning early.